nexong/api/Authentication/authSerializer.py

```diff
@@ -76,6 +76,10 @@ def validate_is_agreed(self, data):
             raise serializers.ValidationError("User must accept terms and conditions.")
         return data
 
+    def validate_has_accepted_latest_terms(self):
+        latest_terms_version = Terms.objects.latest("date").version
+        return self.terms_version_accepted == latest_terms_version
+
```

Comment:
I think the implementation is well done. However, I think this validation would make more sense in `UserSerializer`. This is because, at registration, the terms the user agrees to will be the latest already. The validation should be done when a GET or PUT is done.

Reply:
You are right. I make the changes.